lts-test/src/main/java/com/google/cloud/tools/opensource/lts/LtsCompatibilityTestRunner.java

```diff
@@ -360,4 +360,20 @@ static void modifyBeamGradleFile(Path gradleFile, Bom bom) throws IOException {
           .write(buildGradleContent);
     }
   }
+
+  static void modifyBeamModulePlugin(Path beamModulePluginFile) throws IOException {
+    // We don't want Beam to `force` dependencies when we run tests because the `force` overrides
+    // library versions in the gcp-lts-bom
+    // https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1982#discussion_r611911325
+    String buildGradleContent =
+        Files.asCharSource(beamModulePluginFile.toFile(), StandardCharsets.UTF_8).read();
+
+    buildGradleContent =
+        buildGradleContent.replaceAll(
+            "config.getName\\(\\) != \"errorprone\"",
+            "![\"errorprone\", \"testRuntimeClasspath\"].contains(config.getName())");
```

Comment:
This causes an error where JUnit 4.12 does not have `Assert.assertThrows`, which is available after 4.13.

```
java.lang.NoSuchMethodError: org.junit.Assert.assertThrows(Ljava/lang/Class;Lorg/junit/function/ThrowingRunnable;)Ljava/lang/Throwable;
	at org.apache.beam.sdk.values.PCollectionViewsTest.assertNonEmptyRangesAndPositions(PCollectionViewsTest.java:257)
	at org.apache.beam.sdk.values.PCollectionViewsTest.testNestedOverlaps(PCollectionViewsTest.java:204)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

Reply:
Now `:sdks:java:extensions:google-cloud-platform-core:test` somehow uses Guava's Android flavor. The dependencies task shows no Android flavor.

https://scans.gradle.com/s/t6uughjugn3oy/tests/:sdks:java:extensions:google-cloud-platform-core:test/org.apache.beam.sdk.extensions.gcp.util.GcsUtilTest/testNonExistentObjectReturnsEmptyResult?expanded-stacktrace=WyIwIl0#1

```
java.lang.NoSuchMethodError: com.google.common.cache.CacheBuilder.expireAfterWrite(Ljava/time/Duration;)Lcom/google/common/cache/CacheBuilder;
at com.google.cloud.hadoop.gcsio.GoogleCloudStorageImpl.<init>(GoogleCloudStorageImpl.java:141)	
at org.apache.beam.sdk.extensions.gcp.util.GcsUtil.<init>(GcsUtil.java:201)	
at org.apache.beam.sdk.extensions.gcp.util.GcsUtil.<init>(GcsUtil.java:88)	
at org.apache.beam.sdk.extensions.gcp.util.GcsUtil$GcsUtilFactory.create(GcsUtil.java:110)	
at org.apache.beam.sdk.extensions.gcp.util.GcsUtil$GcsUtilFactory.create(GcsUtil.java:93)	
at org.apache.beam.sdk.options.ProxyInvocationHandler.returnDefaultHelper(ProxyInvocationHandler.java:605)	
at org.apache.beam.sdk.options.ProxyInvocationHandler.getDefault(ProxyInvocationHandler.java:546)	
at org.apache.beam.sdk.options.ProxyInvocationHandler.invoke(ProxyInvocationHandler.java:171)	
at com.sun.proxy.$Proxy41.getGcsUtil(Unknown Source)	
at org.apache.beam.sdk.extensions.gcp.util.GcsUtilTest.testNonExistentObjectReturnsEmptyResult(GcsUtilTest.java:317)	
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```